source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -117,7 +117,7 @@ namespace WorldBuilder
           prm.declare_entry("density", Types::Double(3300),
                             "The reference density of the subducting plate in $kg/m^3$");
 
-          prm.declare_entry("plate velocity", Types::OneOf(Types::Double(0.01),Types::Array(Types::ValueAtPoints(0.01, std::numeric_limits<size_t>::max()))),
+          prm.declare_entry("plate velocity", Types::OneOf(Types::Double(0.01),Types::Array(Types::ValueAtPoints(0.01, std::numeric_limits<uint64_t>::max()))),
```

Comment:
Why do you use uint max for doubles?

Reply:
That contains the max number of elements: https://github.com/GeodynamicWorldBuilder/WorldBuilder/blob/d9c2f64e37b42aa3046ccd9b093830938afaaec9/source/world_builder/types/value_at_points.cc#L31-L33